client/components/Reviews/styles/ReviewsListItem.styled.js

```diff
@@ -1,31 +1,72 @@
 import styled from 'styled-components';
+import Flex from '../../styles/Flex.styled';
+import ButtonLink from '../../styles/ButtonLink.styled';
 
 export const Item = styled.div`
   > * {
-    padding: 10px;
+    margin: 10px 0;
   };
+  padding: 15px 0;
 `;
 
+/* Header */
+
 export const Header = styled.div`
   display: flex;
   justify-content: space-between;
-`;
 
-export const HeaderDetail = styled.div`
-  > span {
-    padding: 4px;
-  };
+  ${Flex} {
+    > span {
+      padding: 0 5px;
+      color: rgb(100, 100, 100);
+    }
+  }
 `;
 
+/* Body */
```

Comment:
Tied to a previous comment: consider adding a body container to reduce the need for comments clarifying what each component is for! You could also consider writing more descriptive variable names (ex: FlexHeader/HeaderFlex) to reduce the need for comments. Only suggestions, code is good as-is for merging.

Reply:
Thanks for pointing this out! I'll consider doing this on a refactor.